src/Event.java

```diff
@@ -0,0 +1,4 @@
+package PACKAGE_NAME;
+
+public class Event {
+}
```

Comment:
Eksempel 2

Reply:
Reel ændring: Vi bør nok ikke kalde vores package for "PACKAGE_NAME", men finde på en anden navngivningsstruktur.